src/shared/shimmer-styles.ts

```diff
@@ -1,10 +1,9 @@
-import { css } from 'leather-styles/css';
-
-export const shimmerStyles = css({
+export const shimmerStyles = {
   '&[data-state=loading]': {
     display: 'inline-block',
     WebkitMask: 'linear-gradient(-60deg, #000 30%, #0005, #000 70%) right/300% 100%',
     backgroundRepeat: 'no-repeat',
     animation: 'shimmer 1.5s infinite',
+    color: 'ink.text-subdued',
   },
-});
+};
```

Comment:
Curious for reason not to pre-wrap in `css`?  If we're not, can we still type the object?

Reply:
not pre-wrapped as we might need to pass smth else to `css`?

Reply:
but that's just more classNames, so ?

`<styled.div className={[shimmerStyles(), css({ ...otherStyles })}].join(' ')` 

I wonder, does panda generate the same css _n_ times if you pass it many times to `css`?

Reply:
just made
```
import type { SystemStyleObject } from 'leather-styles/types';

export const shimmerStyles: SystemStyleObject = {
```